docs/source/app-dev/services.rst

```diff
@@ -0,0 +1,159 @@
+
+The active contract service returns the current contract set as a set of created events that would re-create the state being reported, along with the ledger position at which the view of the set was taken.
+
+For state-driven applications, this is most important at application start. They must synchronize their initial state with a known view of the ledger, and without this service, the only way to do this would be to read the Transaction Stream from the beginning of the ledger. This can be prohibitive with a large ledger.
+
+The active contract service overcomes this, by allowing an application to request a snapshot of the ledger, determine the position at which that snapshot was taken, and build its initial state from this view. The application can then begin to receive events via the Transaction Service from the given position, and remain in sync with the ledger by using these to apply updates to this initial state.
+
+Verbosity
+---------
+
+See :ref:`verbosity` above.
+
+Utility services
+****************
+
+Package service
+===============
+
+Use the **package service** to obtain information about DAML programs and packages loaded into the server.
+
+This is useful for obtaining type and metadata information that allow you to interpret event data in a more useful way.
+
+Ledger identity service
+=======================
+
+Use the **ledger identity service** to get the identity string of the ledger that it is connected to.
+
+You need to include this identity string when submitting commands. Commands with an incorrect identity string are rejected.
+
+Ledger configuration service
+============================
+
+Use the **ledger configuration service** to subscribe to changes in ledger configuration.
+
+This configuration includes maximum and minimum values for the difference in Ledger Effective Time and Maximum Record Time (see `Time Service <#time-service>`__ for details of these).
+
+Time service
+============
+
+Use the **time service** to obtain the time as known by the ledger server.
+
+This is important because you have to include two timestamps when you submit a command - the :ref:`Ledger Effective Time (LET) <com.digitalasset.ledger.api.v1.Commands.ledger_effective_time>`, and the :ref:`Maximum Record Time (MRT) <com.digitalasset.ledger.api.v1.Commands.maximum_record_time>`. For the command to be accepted, LET must be greater than the current ledger time.
+
+MRT is used in the detection of lost commands.
+
+Reset service
+=============
+
+.. ::note
+
+   This is a sandbox feature and not available on production ledgers.
+
+Use the **reset service** to reset the ledger state, as a quicker alternative to restarting the whole ledger application. This is a sandbox feature 
+
+This resets all state in the ledger, *including the ledger ID*, so clients will have to re-fetch the ledger ID from the identity service after hitting this endpoint.
+
+Services diagram
```

Comment:
Why tuck this away down here? I'd put the diagram on the landing page and/or the top of this page.